sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchIndexingBufferedSenderOptions.java

```diff
@@ -21,16 +21,20 @@
  */
 public final class SearchIndexingBufferedSenderOptions<T> {
     private static final boolean DEFAULT_AUTO_FLUSH = true;
-    private static final int DEFAULT_BATCH_SIZE = 500;
+    private static final int DEFAULT_INITIAL_BATCH_ACTION_COUNT = 512;
     private static final Duration DEFAULT_FLUSH_WINDOW = Duration.ofSeconds(60);
-    private static final int DEFAULT_DOCUMENT_TRY_LIMIT = 3;
+    private static final int DEFAULT_MAX_RETRIES = 3;
+    private static final Duration DEFAULT_RETRY_DELAY = Duration.ofMillis(800);
+    private static final Duration DEFAULT_MAX_RETRY_DELAY = Duration.ofMinutes(1);
 
     private final ClientLogger logger = new ClientLogger(SearchIndexingBufferedSenderOptions.class);
 
     private Boolean autoFlush;
     private Duration autoFlushWindow;
-    private Integer batchSize;
-    private Integer documentTryLimit;
+    private Integer initialBatchActionCount;
```

Comment:
Maybe we'd better use property name batchActionCount? 

InitialBatchActionCount is only its initial value, we may change the value when we tune it dynamically.

Reply:
Can change the property name as it doesn't really matter